app/geo/gui/src/ts/twxplore/gui/geo/reducers/map/mapReducer.ts

```diff
@@ -124,9 +123,10 @@ export const mapReducer = (state: MapState, action: BaseAction): MapState => {
         const filterStateOfType =
           result.featureTypesFilters[addedFeature.type!];
         for (const attribute of Object.keys(addedFeature)) {
+          console.log(FeatureAttributeName[attribute as keyof typeof FeatureAttributeName])
+          console.log(FeatureAttribute.AttributeTypeOf(FeatureAttributeName[attribute as keyof typeof FeatureAttributeName]))
           if (
-            new FeatureAttribute(attribute).isNumeric() &&
-            attribute != "postalCode"
+            FeatureAttribute.AttributeTypeOf(FeatureAttributeName[attribute as keyof typeof FeatureAttributeName]).isNumeric()
```

Comment:
Just have the AttributeTypeOf (I suggested above renaming it) take a union type so you can use the string.

type featureAttributeNameOrString = FeatureAttributeName | string
valueOf(name: featureAttributeNameOrString) { if (featureAttributeNameOrString instanceof ...) }